src/util/Options.js

```diff
@@ -69,6 +69,10 @@
  * [User Agent](https://discord.com/developers/docs/reference#user-agent) header
  * @property {PresenceData} [presence={}] Presence data to use upon login
  * @property {IntentsResolvable} intents Intents to enable for this connection
+ * @property {number} [fetchGuildTimeout=15000] Time in milliseconds that clients with the `GUILDS` intent should hang
```

Comment:
```suggestion
 * @property {number} [fetchGuildTimeout=15000] Time in milliseconds that bots with the `GUILDS` intent should hang
```

Shouldn't be this called bots instead of clients?

Reply:
Not at all